libflux/flux-core/src/semantic/nodes.rs

```diff
@@ -1771,6 +1771,53 @@ impl Property {
     }
 }
 
+// #[allow(missing_docs)]
+// pub type Symbol = String;
+
+#[allow(missing_docs)]
+#[derive(Debug, PartialOrd, Ord, PartialEq, Eq, Hash, Clone)]
+pub struct Symbol {
```

Comment:
Does Symbol represent the globally unique identifier?  Or does is represent the locally scoped identifier?

Reply:
I'd expect it to be able to represent either, this is just getting the renaming out of the way.